packages/components/src/Button/index.ts

```diff
@@ -1 +1,2 @@
-export { default, ButtonProps } from "./button";
+export { default } from "./button";
+export type { ButtonProps } from "./button";
```

Comment:
Was seeing a warning when building for Storybook

```
WARNING in ./src/Button/index.ts 1:0-48
"export 'ButtonProps' was not found in './button'
```

Not sure why exporting both at the same time would be problematic.

Maybe TypeScript or Babel wasn't removing the type export as expected, and separating them fixes the warning because we can annotate that ButtonProps is a type (and it's easier to tell that it needs to be removed). That's just a guess, though.